deploy/kubernetes/console/values.yaml

```diff
@@ -13,6 +13,7 @@ dbProvider: mysql
 useLb: false
 console:
   port: 443
+  cookieDomain:
 # externalIP: 127.0.0.1
```

Comment:
Why not set this to '-'

Reply:
Because setting it to -would mean we don't pick up from env.DOMAIN